tests/test_ipld.py

```diff
@@ -151,6 +152,56 @@ def test_transform_cbor_with_nested_link_to_dict():
     assert unmarshal(src) == expected
 
 
+def test_transform_dict_to_cbor_with_multiaddr():
+    addr1 = Multiaddr('/ip4/127.0.0.1/udp/1234')
+    addr2 = Multiaddr('/ipfs/Qmafmh1Cw3H1bwdYpaaj5AbCW4LkYyUWaM7Nykpn5NZoYL')
+
+    src = {
+        'data': 'hello world',
+        'size': 11,
+        'l1': {
+            '/': str(addr1),
```

Comment:
It would be nice if the library would accept raw `Multiaddr` objects inside the source